src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let calls = 0;
+
+  const robot = (n) => {
+    calls++;
+
+    const condition = (calls > 3) && !(calls % 2);
+
+    return condition
+      ? () => 'Bzzz... Error!'
+      : (n2) => n + n2;
```

Comment:
Maybe let's create one arrow function (instead of two) which will return different things?